tests/bcr/MODULE.bazel

```diff
@@ -21,6 +21,10 @@ local_path_override(
 bazel_dep(name = "gazelle", version = "0.33.0")
 bazel_dep(name = "protobuf", version = "3.19.6")
 
+# Required with --incompatible_enable_proto_toolchain_resolution.
+# Avoids building protoc from source, which speeds up CI runs.
+bazel_dep(name = "toolchains_protoc", version = "0.2.1")
```

Comment:
Should we add 2 separate tests: 1 with the pre-built toolchain and 1 with the normal toolchain?

Reply:
I did, see https://github.com/bazelbuild/rules_go/pull/3919/files#diff-b03048100ab56455b5b377a5c20ffd3d17f80c719befca45253af622eb9395efR76.